docs/config/dfdaemon_properties.md

```diff
@@ -0,0 +1,65 @@
+# Customize dfdaemon properties
+
+This topic explains how to cutomize the dragonfly dfdeamon startup parameters. 
+
+**NOTE**: By default, dragonfly config files locate `/etc/dragonfly`. You can create `dfdeamon.yml` for configing dfdaemon startup params. When deploying with Docker, you can mount default path. Properties holds all configurable properties of dfdaemon including `dfget/dfget_server` Properties.
```

Comment:
s/dfdeamon/dfdaemon s/configing/configuring

Reply:
It is better to add an example for how to start dfdaemon in docker with /etc/dragonfly path mounted. WDYT?

Reply:
yep, i will give a example cmd for deploying with Docker.